src/test/munit/body/validation/form/form-body-validation-server.xml

```diff
@@ -22,12 +22,13 @@
       </http:error-response>
     </http:listener>
     <apikit:router config-ref="form-body-validation-config"/>
-    <logger level="INFO" message="#[payload]" />
     <error-handler ref="global-server-error-handler"/>
   </flow>
 
   <flow name="post:\form-data\(form-id)\file:multipart\form-data:form-body-validation-config">
+    <logger level="INFO" message="#[payload]" />
     <set-payload value="#[payload]"/>
+    <logger level="INFO" message="#[payload]" />
```

Comment:
what is the use of the loggers here ?

Reply:
used for testing stream consumption, not necessary now, will be removed